codalab/bundles/run_bundle.py

```diff
@@ -69,7 +69,9 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('exitcode', int, 'Exitcode of the process.', generated=True))
     METADATA_SPECS.append(MetadataSpec('job_handle', str, 'Identifies the job handle (internal).', generated=True, hide_when_anonymous=True))
     METADATA_SPECS.append(MetadataSpec('remote', str, 'Where this job is/was run (internal).', generated=True, hide_when_anonymous=True))
-    METADATA_SPECS.append(MetadataSpec('remotes', list, 'Previous workers where this job was run (internal); multiple values indicate that the bundle was preempted and moved to a different worker.', generated=True, hide_when_anonymous=True))
+    METADATA_SPECS.append(MetadataSpec('remotes', list, 'Previous workers where this job was run (internal); multiple values indicate that the bundle was preempted and moved to a different worker.',
+    generated=True, hide_when_anonymous=True))
+    METADATA_SPECS.append(MetadataSpec('preemptible', bool, 'Whether the bundle is currently running / finished on a preemptible worker.', generated=True, hide_when_anonymous=True, default=False))
```

Comment:
I thought `preemptible` would just specify whether we want to restart the job again if the worker died, but the description here seems to be something else...

Reply:
No -- `preemptible` means that the bundle is currently running / finished on a preemptible worker.

The way we target bundles to run on preemptible workers is to target the bundles using `--tag` to a tag that specifies a preemptible worker pool.

Reply:
@percyliang 
I've added documentation explaining this better:

![image](https://user-images.githubusercontent.com/1689183/144455511-6a96a724-e2a3-4e26-b7b0-c36b3f675e3a.png)

Reply:
I think `on_preemptible_worker` might be a more accurate description, since bundles are not preemptible, workers are.  What do you think?  This is a flag that's automatically set by CodaLab, not by the user.

Reply:
This flag is actually set by the user. We wanted users to explicitly specify `cl run --preemptible` so they are aware that their command must work on preemptible workers. For that reason, I'm going to change it back to `preemptible` instead of `on_preemptible_worker` (because `--on_preemptible_worker` is awkward to type).

Reply:
Never mind, I just removed `--preemptible` entirely from `cl run` and so we can keep calling it `on_preemptible_worker`.